evalml/tests/pipeline_tests/regression_pipeline_tests/test_multiseries_regression_pipeline.py

```diff
@@ -0,0 +1,159 @@
+import pandas as pd
+import pytest
+
+from evalml.pipelines import MultiseriesRegressionPipeline
+from evalml.preprocessing import split_multiseries_data
+
+
+@pytest.fixture(scope="module")
+def component_graph():
+    return {
+        "Time Series Featurizer": ["Time Series Featurizer", "X", "y"],
+        "Baseline Multiseries": [
+            "Multiseries Time Series Baseline Regressor",
+            "Time Series Featurizer.x",
+            "y",
+        ],
+    }
+
+
+@pytest.fixture(scope="module")
+def pipeline_parameters():
+    return {
+        "pipeline": {
+            "time_index": "date",
+            "max_delay": 10,
+            "forecast_horizon": 7,
+            "gap": 0,
+            "series_id": "series_id",
+        },
+        "Time Series Featurizer": {
+            "time_index": "date",
+            "max_delay": 10,
+            "forecast_horizon": 7,
+            "gap": 0,
+            "delay_features": False,
+            "delay_target": True,
+        },
+        "Baseline Multiseries": {"gap": 0, "forecast_horizon": 7},
+    }
+
+
+def test_multiseries_pipeline_init(component_graph, pipeline_parameters):
+    pipeline = MultiseriesRegressionPipeline(component_graph, pipeline_parameters)
+    assert pipeline.series_id == "series_id"
+    assert pipeline.time_index == "date"
+
+
+def test_multiseries_pipeline_invalid_params(component_graph):
```

Comment:
can we add a small test case to confirm that the multiseries pipeline errors out if the time series featurizer isn't used?

Reply:
Only the baseline regressor requires a time series featurizer, not the pipeline as a whole (and I mostly used the baseline for testing because it's the only multiseries estimator that's in main right now lol) - there's tests for it within the baseline itself, I'm not sure it makes sense to add a test for it in the pipeline as well